lib/shopify_api/auth/session.rb

```diff
@@ -121,6 +121,19 @@ def deserialize(str)
         end
       end
 
+      sig { params(other: Session).returns(Session) }
+      def copy_attributes_from(other)
```

Comment:
Is this method equivalent to `serialize` + `deserialize`?

Reply:
I considered using serialize/deserialize but `deserialize` is a class method that creates a new object, and we need to mutate the existing one.